src/org/usfirst/frc4904/robot/humaninterface/drivers/DefaultDriver.java

```diff
@@ -10,7 +12,10 @@ public DefaultDriver() {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		AutoShifter shiftCommand = new AutoShifter();
```

Comment:
If this is inside the driver, why is it also in Robot.java? (I recommend removing it from Robot.java)